nnpdf_data/nnpdf_data/new_commondata/STAR_2012_2JET_B_510GEV/uncertainties.yaml

```diff
@@ -0,0 +1,887 @@
+definitions:
+  stat:
+    description: statistical uncertainty
+    treatment: ADD
+    type: UNCORR
```

Comment:
Are we sure that we have to retain this stat uncertainty? Reading the paper, my understanding is that the provided correlation matrices have to be multiplied by the total uncertainty, which is the sum in quadrature of the statistic and systematic uncertainties, for jet-jet, jet-dijet and dijet-dijet.